docs/version0.21.md

```diff
@@ -48,6 +49,30 @@ To learn more about support for OpenJ9 releases, including OpenJDK levels and pl
 
 Feature details............
 
+### Update to `NoClassDefFoundError` exception message
+
+The class names printed in a `NoClassDefFoundError` exception message have been updated to match the same order as reported by Hotspot.
+
+For example, given a class `Main.java`
+
+```
+package mypackage;
+
+public class Main {
+	public static void main(String [] args) {
+		System.out.println("Hello!");	
+	}
+}
+```
+Compiling and running this program in the `mypackage` directory will result in the following error:
+```
+root@mycomputer:~/path/to/mypackage javac Main.java
+root@mycomputer:~/path/to/mypackage java Main
+Error: Could not find or load main class Main
+Caused by: java.lang.NoClassDefFoundError: mypackage/Main (wrong name: Main)
```

Comment:
By an example, I just meant this line and the following paragraph. I don't think the code example to recreate it is necessary, but it doesn't need to be removed unless the docs team thinks it's too much for a What's New.